packages/bodiless-core/src/withResetButton.ts

```diff
@@ -13,13 +13,13 @@
  */
 
 import { flowHoc, HOC } from '@bodiless/fclasses';
+import { useNode } from '@bodiless/data';
 import {
   withContextActivator,
   withLocalContextMenu,
-} from '../hoc';
-import { withMenuOptions } from '../PageContextProvider';
-import { useNode } from '../NodeProvider';
-import { TMenuOption } from '../Types/ContextMenuTypes';
+} from './hoc.bl-edit';
```

Comment:
you think it's nt worth creating a static version of this which is just "identity"?

Reply:
yes, I skipped it because it was not included in __vital__, but it is in vital-demo, so better to create the static version, done